README.md

```diff
@@ -122,6 +122,41 @@ You can build a docker image locally with the following commands:
 
 *NB* if you are on a Mac, you will need [gnu-tar](https://formulae.brew.sh/formula/gnu-tar).
 
+## Using Prometheus as a Go Library
+
+### Remote Write
+
+We are publishing our Remote Write proto independently at
+[buf.build](https://buf.build/prometheus/prometheus/assets).
+
+You can use that as a library:
+
+```shell
+$ go get go.buf.build/protocolbuffers/go/prometheus/prometheus
+```
+
+This is experimental.
+
+### Prometheus code base
+
+We try to minimize the amount of code in our codebase that is not relevant to
+our purpose. However, we appreciate other projects building up on our code and
+reusing our codebase as libraries.
```

Comment:
I'm not sure if this preamble is really helpful. I appreciate the intention to provide some context, but I think it had to be explained in more detail to be useful. But then it would be so long that it might be better to just focus on what has to be done.